applications/portal/backend/api/services/user_service.py

```diff
@@ -0,0 +1,18 @@
+from cloudharness.middleware import get_authentication_token
+from cloudharness_django.models import Team, Member
+from django.contrib.auth.models import User
+import jwt
+
+def get_current_user():
```

Comment:
@D-GopalKrishna for Django based apps we don't need to query CloudHarness

you can just grab the user from the request